### PR TITLE
New proposed text for main sections

### PR DESCRIPTION
**Pull request by littledan**

### CODE REVIEW

File: _site/index.html [hunk 1]
```diff
@@ -92,30 +92,26 @@ <h3>
             Who?
           </h3>
           <p>
-            The specification is standardized by the TC39 (Technical committee 39) of ECMA (European
-            Computer Manufacturers Association). You can read our <a href="https://ecma-international.org/memento/TC39.htm#Scope">the scope of our work here</a>
+            TC39 is a body of JS developers, implementers, academics and more, collaborating with the community to maintain and evolve the definition of JS. We are part of <a href="http://www.ecma-international.org/">Ecma International</a>.
```
Comment (xtuc) [inline _site/index.html]: As a non native English speaker; I don't know what "body of JS developers" means. Could we rephrase it to group? or something equivalent.

**Review by xtuc — COMMENTED** [review]
Looks good!

Could we also in general use JavaScript instead of JS?

File: _site/index.html (hunk 1 above)
Comment (ljharb) [reply _site/index.html]: This needs an Oxford comma after “academics”

File: _site/index.html [hunk 2]
```diff
@@ -92,30 +92,26 @@ <h3>
...
         </span>
         <span class="basic-info">
           <h3>
             How?
           </h3>
           <p>
-          We have a <a href="https://tc39.github.io/process-document/">four stage process</a> with a
-          rolling release schedule with a new specification published every year.</p> <p>Proposals move
-          through stages at TC39 meetings, which are held every <a href="https://github.com/tc39/agendas/">two months</a>, and the <a href="https://github.com/tc39/tc39-notes/">Notes</a> are made public.
+            We develop the ECMAScript specification <a href="https://github.com/tc39/ecma262/">on GitHub</a> and at meetings every two months. We follow a <a href="https://tc39.github.io/process-document/">four stage process</a> for new <a href="https://github.com/tc39/proposals/">language feature proposals</a>. See our <a href="https://github.com/tc39/agendas/">meeting agendas</a> and <a href="https://github.com/tc39/tc39-notes/">minutes</a> to learn more.
```
Comment (ljharb) [inline _site/index.html]: Does “ecmascript” need defining first?

File: _site/index.html [hunk 3]
```diff
@@ -92,30 +92,26 @@ <h3>
...
           <p>
-  There are many opportunities to get involved on an individual level. Check out our
-  <a href="(https://github.com/tc39/proposals/blob/master/CONTRIBUTING.md">Contributor Documentation</a> and our guides to learn more.
-
-  To get involved with the committee on an organizational level, read our
-  <a href="https://ecma-international.org/memento/TC39-M.htm">Joining the TC39</a> documentation.
+            TC39 welcomes contributions from the JS community, whether it is feedback on existing proposals, improved documentation, testing, implementations, or even language feature ideas. See our
+  <a href="https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md">contributor guide</a> for details.
+          </p><p>
+            To participate as a TC39 delegate in meetings, <a href="https://ecma-international.org/memento/join.htm">join Ecma</a> and our <a href="https://ecma-international.org/memento/Policies/Registration%20Form%20for%20the%20Ecma%20Royalty%20Free%20(RF)%20Task%20Group%20(TG).pdf">patent agreement</a>.
```
Comment (ljharb) [inline _site/index.html]: and *sign* our?

File: _site/index.html (hunk 1 above)
Comment (littledan) [reply _site/index.html]: @xtuc Would the word "group" make more sense?

@ljharb I didn't realize all TC39 web properties needed to follow UK English style. Is that the case? Do you think we need to use UK spellings?

Comment (ljharb) [reply _site/index.html]: The Oxford comma isn't a UK style, it's an unambiguous style used throughout the US - one that is used elsewhere in the document.

Comment (ljharb) [reply _site/index.html]: An amusing US court case where the lack of an Oxford comma cost a company millions: https://www.nytimes.com/2018/02/09/us/oxford-comma-maine.html

**Comment by littledan** [comment]
I don't know what to think about the terms ECMAScript, JavaScript, and JS. There's a few things floating around:
- The standard consistently says ECMAScript. It would be a little weird to not draw the connection to that. I think there's some understanding in the JS community that JavaScript is "technically" called ECMAScript, right?
- JavaScript is a trademark in a funny sort of state
- JS is not trademarked
- I guess JavaScript would be the most intelligible, and second JS, third ECMAScript. Is that right?

Maybe the homepage should use both ECMAScript and either JS or JavaScript (depending what's OK), and we should make sure the association is clearly drawn.

**Comment by ljharb** [comment]
I think we should define ECMAScript once, but otherwise refer to it as JavaScript.

File: _site/index.html (hunk 1 above)
Comment (littledan) [reply _site/index.html]: Wow, that missing comma is definitely not an Oxford comma. Anyway, I'll insert the commas here.

**Comment by iddan** [comment]
Is it ready to be merged?

**Review by ljharb — APPROVED** [review]

File: _site/index.html [hunk 4]
```diff
@@ -92,30 +92,26 @@ <h3>
...
           <p>
-  There are many opportunities to get involved on an individual level. Check out our
-  <a href="(https://github.com/tc39/proposals/blob/master/CONTRIBUTING.md">Contributor Documentation</a> and our guides to learn more.
-
-  To get involved with the committee on an organizational level, read our
-  <a href="https://ecma-international.org/memento/TC39-M.htm">Joining the TC39</a> documentation.
+            TC39 welcomes contributions from the JavaScript community, whether it is feedback on existing proposals, improved documentation, testing, implementations, or even language feature ideas. <!-- TODO: Make each of the things in the preceding sentence links to the appropriate guides -->
+            See our <a href="https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md">contributor guide</a> for details. 
+          </p><p>
+            To participate as a TC39 delegate in meetings, <a href="https://ecma-international.org/memento/join.htm">join Ecma</a> and sign our <a href="https://ecma-international.org/memento/Policies/Registration%20Form%20for%20the%20Ecma%20Royalty%20Free%20(RF)%20Task%20Group%20(TG).pdf">patent agreement</a>.
```
Comment (codehag) [inline _site/index.html:114]: in a future iteration, we should have delegate as a definable word!

Comment (littledan) [reply _site/index.html:114]: Should we reword this as, "To participate in physical TC39 meetings" ?

Comment (ljharb) [reply _site/index.html:114]: This includes when you’re not physically present for a regular meeting and dial in tho

Comment (littledan) [reply _site/index.html:114]: OK, let's just say, "To participate in TC39 meetings", then.

**Comment by littledan** [comment]
@allenwb It'd be great to have your review of this text.

**Comment by xtuc** [comment]
Thanks for the updates @littledan
